packages/ui/src/app/pages/Election/components/ElectionProgressBar.tsx

```diff
@@ -27,11 +27,12 @@ interface ElectionProgressBarProps extends StatisticItemProps {
   electionStage: string
 }
 
+const GENESIS_BLOCK_TIMESTAMP = 1670693046000
+
 const blockDurationToMs = (blockDuration: number) => blockDuration * MILLISECONDS_PER_BLOCK
-const blockToDate = (duration: number) => {
-  const now = Date.now()
-  const msDuration = blockDurationToMs(duration)
-  return new Date(now + msDuration).toLocaleString('en-gb', { timeZone: 'Europe/Paris' })
+const blockToDate = (block: number) => {
+  const msDuration = blockDurationToMs(block)
+  return new Date(GENESIS_BLOCK_TIMESTAMP + msDuration).toLocaleString('en-gb', { timeZone: 'Europe/Paris' })
```

Comment:
Finally here's how to remove the seconds:
```suggestion
  const date = new Date(now + msDuration)
  return `${date.toLocaleString('en-gb', { timeZone: 'Europe/Paris', dateStyle: "short", timeStyle: "short" })} CET` 
```
It's closer to the original design and it's a lot better since the time estimation can't be accurate to the second.